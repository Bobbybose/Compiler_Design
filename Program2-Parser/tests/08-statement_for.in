for (i=0; i<a; i = i +1) j = j + 1;
