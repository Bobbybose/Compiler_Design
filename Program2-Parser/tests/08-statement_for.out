statementList: 1,1
|   
+-- statementList (EMPTY STRING): 1,1
|   
+-- statement: 1,1
|   |   
|   +-- for: 1,1
|   |   
|   +-- (: 1,5
|   |   
|   +-- expression: 1,6
|   |   |   
|   |   +-- expression: 1,6
|   |   |   |   
|   |   |   +-- i: 1,6
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- =: 1,7
|   |   |   
|   |   +-- expression: 1,8
|   |   |   |   
|   |   |   +-- 0: 1,8
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   *   
|   |   
|   +-- ;: 1,9
|   |   
|   +-- expression: 1,11
|   |   |   
|   |   +-- expression: 1,11
|   |   |   |   
|   |   |   +-- i: 1,11
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- <: 1,12
|   |   |   
|   |   +-- expression: 1,13
|   |   |   |   
|   |   |   +-- a: 1,13
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   *   
|   |   
|   +-- ;: 1,14
|   |   
|   +-- expression: 1,16
|   |   |   
|   |   +-- expression: 1,16
|   |   |   |   
|   |   |   +-- i: 1,16
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- =: 1,18
|   |   |   
|   |   +-- expression: 1,20
|   |   |   |   
|   |   |   +-- expression: 1,20
|   |   |   |   |   
|   |   |   |   +-- i: 1,20
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   +-- +: 1,22
|   |   |   |   
|   |   |   +-- expression: 1,23
|   |   |   |   |   
|   |   |   |   +-- 1: 1,23
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   *   
|   |   
|   +-- ): 1,24
|   |   
|   +-- statement: 1,26
|   |   |   
|   |   +-- expression: 1,26
|   |   |   |   
|   |   |   +-- expression: 1,26
|   |   |   |   |   
|   |   |   |   +-- j: 1,26
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   +-- =: 1,28
|   |   |   |   
|   |   |   +-- expression: 1,30
|   |   |   |   |   
|   |   |   |   +-- expression: 1,30
|   |   |   |   |   |   
|   |   |   |   |   +-- j: 1,30
|   |   |   |   |   |   
|   |   |   |   |   *   
|   |   |   |   |   
|   |   |   |   +-- +: 1,32
|   |   |   |   |   
|   |   |   |   +-- expression: 1,34
|   |   |   |   |   |   
|   |   |   |   |   +-- 1: 1,34
|   |   |   |   |   |   
|   |   |   |   |   *   
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- ;: 1,35
|   |   |   
|   |   *   
|   |   
|   *   
|   
*   
